include/openthread/link.h

```diff
@@ -213,23 +213,27 @@ OTAPI otError OTCALL otLinkSetExtendedAddress(otInstance *aInstance, const otExt
 OTAPI void OTCALL otLinkGetFactoryAssignedIeeeEui64(otInstance *aInstance, otExtAddress *aEui64);
 
 /**
- * This function returns the maximum transmit power setting in dBm.
+ * This function returns the radio's transmit power configuration
+ *
+ * The 32-bit value's interpretation is radio-specific.
  *
  * @param[in]  aInstance   A pointer to an OpenThread instance.
  *
- * @returns  The maximum transmit power setting.
+ * @returns  The transmit power configuration.
  *
  */
-OTAPI int8_t OTCALL otLinkGetMaxTransmitPower(otInstance *aInstance);
+OTAPI uint32_t OTCALL otLinkGetTransmitPower(otInstance *aInstance);
```

Comment:
Suggestion: how about changing them to `Get/SetTransmitPowerConfiguration`...I know the name is getting long, but I feel it'd be good to emphasize that it's a (radio specific) configuration value and not tx power in dBm.